webapp/app.py

```diff
@@ -208,46 +210,53 @@ def remove_webpage(body: RemoveWebpageModel):
             jira_tasks = JiraTask.query.filter_by(webpage_id=webpage_id).all()
             if jira_tasks:
                 for task in jira_tasks:
-                    if app.config["JIRA"].change_issue_status(
+                    status_change = app.config["JIRA"].change_issue_status(
                         issue_id=task.jira_id,
                         transition_id=JiraStatusTransitionCodes.REJECTED.value,
-                    ):
-                        JiraTask.query.filter_by(id=task.id).delete()
+                    )
+                    if status_change["status_code"] != 204:
```

Comment:
We already check for this in `self.__request__`, not so?  So maybe we should check another field for validity of this request.

Reply:
This is the only thing that is returned by Jira change status API

Reply:
I see. In that case it's alright